paasta_tools/mesos_tools.py

```diff
@@ -468,10 +469,15 @@ def get_mesos_state_from_leader():
     return state
 
 
-def get_mesos_quorum(state):
+def get_mesos_state_summary_from_leader():
+    res = master.CURRENT.fetch("/master/state-summary")
```

Comment:
I'm kinda surprised this works. The actual endpoint is "state/summary.json". Does mesos-cli does some sort of translation?